integration

```diff
@@ -751,6 +751,7 @@
   "ReneNulschDE/mbapi2020",
   "rexave/hass-orange-internet-on-the-move",
   "rgc99/irrigation_unlimited",
+  "rlippmann/hass-adtpulse",
```

Comment:
I'm confused, this already exist?
And has no changes on top of the existing.
<https://github.com/rsnodgrass/hass-adtpulse>

Reply:
It exists, but I've been making changes in my repo and merging them into Ryan's repo until the change of ownership goes through.

Reply:
Then Im more confused, this is not how changing ownership works on GitHub.
What you have done with that repo now is a fork, not a ownership transfer, you can look more here https://docs.github.com/en/repositories/creating-and-managing-repositories/transferring-a-repository

Reply:
It started out as a fork about a year ago.  Then the original author decided he wasn't interested in working on it anymore.  So he gave me admin rights on the repo.  If I need to have him just assign me ownership instead, i'll do that.  Unfortunately, since I've evidently already made a fork, I can't rename the repo.  Or can I just delete my fork?

Reply:
I would advise against deleting things.
If you need help with renaming and or transferring I suggest looking at the github documentation or their support.

Reply:
So, I would have the owner give me ownership of his repo, archive my fork, make changes there, and just close this PR?

Sorry, I'm a HACS/github n00b. :)

Reply:
I can not comment on that as I have no affiliation with that repo.